docs/data/query.md

```diff
@@ -112,48 +112,38 @@ You do not need to perform any additional configuration when [querying data in t
 
    This command configures a new database user for your org for use with data query.
 
-1. Determine the hostname for your organization's MongoDB Atlas Data Federation instance by running the following command with the organization's `org-id` from step 2:
+1. Determine the connection URI for your organization's MongoDB Atlas Data Federation instance by running the following command with the organization's `org-id` from step 2:
 
    ```sh {class="line-numbers linkable-line-numbers"}
    viam data database hostname --org-id=<YOUR-ORGANIZATION-ID>
    ```
 
-   This command returns the `hostname` (including database name) to use to connect to your data store on the Viam organization's MongoDB Atlas instance.
+   This command returns the hostname (including database name) and connection URI to use to connect to your data store on the Viam organization's MongoDB Atlas instance.
+   The connection URI includes everything you need to connect to your organization's MongoDB Atlas Data Federation instance from a valid MQL-compatible client, except your Viam account password.
```

Comment:
(here and elsewhere) the password required for connection will be the configured DB password (via `viam data database configure`), _not_ the user account's password. Saying "Viam account password" might be confusing?

Reply:
@n0nick Whoops, you're right, thank you -- fixed!